### DISCUSSION TITLE
should I use node every Time with admin rights

### DISCUSSION THREAD

**Discussion by niusz** [coreybutler/nvm-windows#602] [Support] [answered]
After I install nvm ,I find when I run `npm run serve ` in my project ,it remind me node is not an internal or external command. is that means I should use admin modle when I run my project every time?

**Comment by coreybutler** [comment, marked as answer]
Not necessarily. It depends on where your files exist. Running npm doesn't touch NVM4W at all, so in this regard, npm will behave the same way as if you'd installed Node without NVM4W.

Windows has several protected directories. For example, `C:\Program Files` is one of them. These directories are often restricted to user accounts with admin rights. If you work in a company that locks down other directories through Group Policy, then you may need special access on those directories as well. In your situation, it may be as easy as moving your code to a different directory, like `C:\my_project`.

Please keep in mind that if you're using the latest version of node, the versions of npm that have been shipping have had a few issues. It's possible you may just need to update npm (`npm install -g npm`).

While it may be obvious, it's also worth mentioning that you need to run `nvm use x.x.x`, where `x.x.x` is the version of Node you want to run. When npm says node is not an external command, that usually means it cannot find node. The most common reason for that is `nvm install` was run, but not `nvm use`. The `nvm use` command is what sets the symlink (i.e. links to Node).

Without knowing more about the circumstance, it's hard to troubleshoot... but the bottom line is you do not need to have administrative privileges to run Node. You may need an admin account to satisfy your own system permission requirements, but on a stock install of NVM4W, you only need admin privileges to run `nvm` commands, not `node` or `npm`.

**Reply by niusz** [reply]
Thank you very much, because my pool English,I can't write more.
